File: ClientApp/src/models/Programaciones.ts
export class Programaciones{
    constructor(
        public id:number,
        public codigoEquipo:string,
        public idEquipo:number,
        public descEquipo:string,
        public ano:number,
        public semana:number,
        public fecha:Date,
        public idEstado:number,
        public descEstado:string,        
        public idUsuario:number,
        public nombreUsuario:string,
        public observaciones:string,
        public frecuencia:number,
        public idArea:number,
        public mes:number
    ){

    }
}

File: Database/Programaciones.cs
using System;
using System.Collections.Generic;

namespace MetAndsInsUn.Database
{
    public partial class Programaciones
    {
        public long Id { get; set; }
        public long? IdEquipo { get; set; }
        public int? Ano { get; set; }
        public int? Semana { get; set; }
        public long? IdEstado { get; set; }
        public DateTime? Fecha { get; set; }
        public long? IdUsuario { get; set; }
        public string Observaciones { get; set; }

        public virtual Equipos IdEquipoNavigation { get; set; }
        public virtual TiposData IdEstadoNavigation { get; set; }
        public virtual Usuarios IdUsuarioNavigation { get; set; }
    }
}
